.github/workflows/ci.yml

```
@@ -19,18 +19,13 @@ jobs:
       matrix:
         runs-on: ["ubuntu-latest", "windows-latest"] # can add macos-latest
         python-version: ["3.11", "3.12", "3.13"]
-        test-config:
-          - tests-path: "tests/unit_tests"
-            needs-services: false
-          - tests-path: "tests/system_tests"
-            needs-services: true
+        tests-path: ["tests/unit_tests", "tests/system_tests"]
```

We lost the `docs` and `src` check somewhere previously by explicitly passing unit tests, we can restore it by not passing anything:
```suggestion
        tests-path: ["", "tests/system_tests"]
```